src/lib/arch/aarch64/InstructionMetadata.cc

```diff
@@ -1110,6 +1478,9 @@ void InstructionMetadata::revertAliasing() {
         operands[3].imm = 31;
         return;
       }
+      if (opcode == Opcode::AArch64_SXTW_ZPmZ_D) {
+        return;
```

Comment:
Blank entry for same reason as AArch64_MUL_ZPmZ_{B,D,H,S} instructions?

Reply:
yes, I forgot to add the comment to this one - have done so now